test/suite/test_sweep04.py

```diff
@@ -0,0 +1,248 @@
+                # the loop, to give the various sweeps time to operate.
+                thistime = time.time()
+                delta = thistime - lasttime
+                if delta < 3.0:
+                    time.sleep(3.0 - delta)
+                lasttime = thistime
+
+            if stressing:
+                self.session.drop(self.transient_uri(available_transient), "force")
+                available_transient += 1
+                self.create_table(self.transient_uri(created_transient))
+                created_transient += 1
+
+            rand_session = sessions[r.rand_range(0, self.nsessions)]
+
+            # In the stress part of the run, some small fraction (given by ratio_examined) will
+            # look at the transient tables.  Looking at these rarely makes them candidates for
+            # closing by the connection sweep.
+            big = 1000000   # Any large number works.
+            if stressing and r.rand32() % big > big * self.ratio_examined:
+                # Access "count" transient tables, starting at table "tnum".
+                count = self.transient_examined
+                tnum = r.rand_range(available_transient, available_transient + self.transient_tables - count)
+                self.examine(rand_session, self.transient_uri, tnum, count)
+            else:
+                # Access a single core table, numbered "tnum".
+                tnum = r.rand_range(0, self.core_tables)
+                self.examine(rand_session, self.core_uri, tnum, 1)
+
+            if loopcount % 100 == 99:
+                # Gather statistics about the number of dhandles, to be checked at
+                # the end of the run.
+                stat_cursor = self.session.open_cursor('statistics:', None, None)
+
+                # Enable for detailed output.
+                if False:
+                    close = stat_cursor[stat.conn.dh_sweep_close][2]
+                    remove = stat_cursor[stat.conn.dh_sweep_remove][2]
+                    sweep = stat_cursor[stat.conn.dh_sweeps][2]
+                    sclose = stat_cursor[stat.conn.dh_session_handles][2]
+                    ssweep = stat_cursor[stat.conn.dh_session_sweeps][2]
+                    tod = stat_cursor[stat.conn.dh_sweep_tod][2]
+                    ref = stat_cursor[stat.conn.dh_sweep_ref][2]
+                    self.pr(('DHANDLE STATS: close={}, remove={}, sweep={}, session_handles={}, '+
+                             'session_sweeps={}, sweep_tod={}, sweep_ref={}').format(
+                                 close, remove, sweep, sclose, ssweep, tod, ref))
+
+                dhandles = stat_cursor[stat.conn.dh_conn_handle_count][2]
+                files_open = stat_cursor[stat.conn.file_open][2]
+                self.pr('  dhandle_count={}'.format(dhandles))
+                self.pr('  file_open={}'.format(files_open))
+                dhandle_counts.append(dhandles)
+                stat_cursor.close()
+                if loopcount % 100 == 999:
```

Comment:
Typo? `loopcount % 100` will never be `999`

Reply:
Thanks, I'll fix!

Reply:
Ha, 'fixing' it causes the test to fail, as it writes to stdout, and that fails the test.  So I fixed, but disabled with a comment.

Reply:
👍